### PR TITLE
Format Numbers of Dashboard

### PR DESCRIPTION
**Pull request by seanmccann**
This will format the numbers on the dashboard by adding a comma for thousands, and will scale the number using css.

Examples:
- 99 => 99
- 9999 => 9,999
- 10200 => 10,200

### CODE REVIEW

**Comment by aballman** [comment]
@seanmccann What was the motivation for submitting this? Are you using the dashboard in a way that prevents from displaying the full number? We sort of like getting an exact count out of the dashboard.

We would be amenable to this, if you really want it, if you include a tooltip on hover for displaying the exact count.

**Comment by seanmccann** [comment]
screen shot 2018-11-30 at 12 07 31 pm

@aballman Because it's cut off. We could just make the font smaller too.

**Comment by aballman** [comment]
That would be a good reason! Do you usually use CodeAmp in a smaller window? Amos and I tried it in a few different sizes and didn't get that same scenario until the window was very small.

**Comment by aballman** [comment]
@seanmccann If you conditionally enable this when an overflow condition occurs we'll take it. It looks like you could use this lib to accomplish this: https://github.com/nickuraltsev/react-overflow

File: src/components/Dashboard/style.module.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
 }
 
 .bigNumber {
-  font-size: 80px;
+  font-size: 4.5vw;
```
Comment (seanmccann) [inline src/components/Dashboard/style.module.css]: This is 4.5% of the viewport width, which works really great for auto scaling

**Comment by seanmccann** [comment]
@aballman I change it so that it not longer shortens numbers. It now just adds a comma (1000 => 1,000) and uses CSS to auto scale the number size to fit inside the box.

**Comment by aballman** [comment]
Somehow this seemed to have increased the size of the boxes as well, probably resized to fit the larger  text size? I'll take it either way but I am curious.

**Pull request merged**